22:29:11.300 INFO Test execution started: test_page/add_description
22:29:11.301 INFO Browser: chrome
22:29:11.301 INFO Environment: test
22:29:11.301 INFO Using data: 
    description: description of the test
    url: http://localhost:8000

22:29:13.410 INFO Navigate to: 'http://localhost:8000'
22:29:13.520 INFO Write 'admin' in element username
22:29:13.598 INFO Write 'admin' in element password
22:29:13.641 INFO Click login_button
22:29:14.187 INFO Random value generated: tsuetva
22:29:14.188 INFO Store value tsuetva in key name
22:29:15.479 INFO Write 'tsuetva' in element None
22:29:15.480 INFO Press key: ENTER
22:29:15.575 INFO Click None
22:29:15.874 INFO Write 'description of the test' in element description
22:29:15.909 INFO Click save
22:29:15.970 INFO Waiting for 5 seconds
22:29:21.040 INFO Refresh page
22:29:21.096 INFO Refresh page
22:29:21.157 INFO Refresh page
22:29:21.217 INFO Refresh page
22:29:21.217 INFO Waiting for 10 seconds
22:29:31.253 INFO Verify element 'description' contains text 'description of the test'
22:29:31.270 INFO Take screenshot error
22:29:31.624 ERROR An error ocurred:
Traceback (most recent call last):
  File "/Users/lucianofeo/Desktop/golem/src/golem/test_runner/test_runner.py", line 115, in run_test
    test_module.test(execution.data)
  File "/Users/lucianofeo/Desktop/golem/example/projects/golem/tests/test_page/add_description.py", line 25, in test
    verify_text_in_element(test_page.description, data.description)
  File "/Users/lucianofeo/Desktop/golem/src/golem/actions.py", line 372, in verify_text_in_element
    .format(text, webelement.name))
golem.core.exceptions.TextNotPresent: Text 'description of the test' was not found in element description
22:29:31.624 INFO Close driver
